docs/docs/reference-react.md

```diff
@@ -73,7 +81,13 @@ class Greeting extends React.Component {
 }
 ```
 
-`React.Component` is the base class for React components when they are defined using [ES6 classes](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Classes). See the [React.Component API Reference](/react/docs/react-component.html) for a list of methods related to the base `React.Component` class.
+See the [React.Component API Reference](/react/docs/react-component.html) for a list of methods and properties related to the base `React.Component` class.
+
+* * *
+
+### `React.PureComponent`
+
+`React.PureComponent` is exactly like [`React.Component`](#react.component) but implements [`shouldComponentUpdate()`](/react/docs/react-component.html#shouldcomponentupdate) with a shallow prop and state comparison. It is equivalent to a class with [`PureRenderMixin`](/react/docs/pure-render-mixin.html).
```

Comment:
Sorry I should've been more clear. We should explain what it does (can reuse explanation from that page). We should not point people to a page about mixin because mixin are legacy, docs don't prominently mention them anymore, and people shouldn't have to know what they are to understand how and when to use PureComponent.